helm-charts/sawmills-collector/tests/keda_scaler_address_test.yaml

```
@@ -72,3 +72,25 @@ tests:
+      keda.scaling.external.loadBalancerMetadata.query: sum(otelcol_loadbalancer_central_queue_compressed_bytes)
+      keda.scaling.external.loadBalancerMetadata.targetValue: "10485760"
+      kedaScaler.enabled: true
+      kedaScaler.service.monitoringPort: 19466
+      loadBalancer.enabled: true
+    asserts:
+      - equal:
+          path: spec.triggers[0].metadata.url
+          value: http://sawmills-collector-keda-otel-scaler.sawmills-o11y.svc.cluster.local:19466/query?query=sum%28otelcol_loadbalancer_central_queue_compressed_bytes%29
+      - notMatchRegex:
+          path: spec.triggers[0].metadata.url
+          pattern: '\.sawmills\.svc\.cluster\.local'
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)